### ISSUE TITLE
请教下全连接层输出的结果是怎么计算的。

### ISSUE THREAD

**Issue by sunlaobo** [lllcho/CAPTCHA-breaking#5]
model = Sequential()
model.add(Convolution2D(32, 1, 4, 4, border_mode='full', activation='relu'))
model.add(Convolution2D(32, 32, 4, 4, activation='relu'))
model.add(MaxPooling2D(poolsize=(3, 3)))
model.add(Dropout(0.25))
model.add(Convolution2D(64, 32, 4, 4, border_mode='full', activation='relu'))
model.add(Convolution2D(64, 64, 4, 4, activation='relu'))
model.add(MaxPooling2D(poolsize=(2, 2)))
model.add(Dropout(0.25))
model.add(Flatten())
model.add(Dense(64 \* 5 \* 5, 512, activation='relu'))
model.add(Dropout(0.5))
model.add(Dense(512, nb_class, activation='softmax'))
model.load_weights(model_path)
# model.compile(loss='categorical_crossentropy', optimizer='adagrad')

请教下上面的model.add(Dense(64 \* 5 \* 5, 512, activation='relu'))中的5*5是怎么计算出来的。小白，先谢过

**Comment by lllcho** [comment]
根据输入层图像的大小，逐层计算。border_mode='full'，当kernel-size为4*4时，会将输入在宽高两个方向上分别增加3；Maxpooling层除以poolsize(当除不尽时向下取整)
